src/test/java/fiori/component/TabContainer.java

```diff
@@ -21,4 +23,32 @@ public TabContainer(By locator) {
     public WebElement getActiveTab() {
         return getElement().findElement(By.className("th-l-anchorNav-topAB"));
     }
+
+    /**
+     * The method returns the tab from the tab container by its name, passed as a parameter.
+     *
+     * @param tabName - the name of the tab to return
+     * @return WebElement tab
+     */
+    private WebElement getTab(String tabName) {
+        for (WebElement tab : getElement().findElements(By.className("th-bt-b"))) {
+            if (tab.getText().equals(tabName)) {
+                return tab;
+            }
+        }
+        return null;
```

Comment:
It's better to return "throw new IllegalStateException("Cannot locate tab with label [" + tabName + "]");" instead null